src/Metadata/Resource/ResourceMetadata.php

```diff
@@ -163,6 +165,31 @@ public function withCollectionOperations(array $collectionOperations): self
     }
 
     /**
+     * Gets subresource operations.
+     *
+     * @return array|null
+     */
+    public function getSubresourceOperations()
+    {
+        return $this->subresourceOperations;
+    }
+
+    /**
+     * Returns a new instance with the given subresource operations.
+     *
+     * @param array $subresourceOperations
```

Comment:
You can remove those annotations (it's already documented by typehints).

Reply:
copy pasted, why don't we remove the other ones then?